lambdas/navigator_config/index.py

```diff
@@ -1,24 +1,43 @@
+"""
+navigator-config
+
+Initializes and updates catalog config files
+"""
 import json
 
 import boto3
 from botocore.exceptions import ClientError
 import cfnresponse
 
-s3_client = boto3.client('s3')
+S3_CLIENT = boto3.client('s3')
+
+SENTRY_DSN = "https://cfde44007c3844aab3d1ee3f0ba53a1a@sentry.io/1410550"
 
 def handler(event, context):
+    """
+    top-level handler for CloudFormation custom resource protocol
+    """
     if event['RequestType'] == 'Delete':
         cfnresponse.send(event, context, cfnresponse.SUCCESS, {})
         return
 
+    props = event['ResourceProperties']
+
+    if event['RequestType'] == 'Create':
+        # stack create
+        try:
+            create_configs(**props)
+            cfnresponse.send(event, context, cfnresponse.SUCCESS, {})
+            return
+        except Exception:
+            cfnresponse.send(event, context, cfnresponse.FAILED, {})
+            raise
+
+    # event['RequestType'] == 'Update'
```

Comment:
Turn this into a sentences so it's clear that you're documenting and this isn't an artifact of debugging
```
# In other cases try and update, i.e. when event['RequestType'] == 'Update'
```